src/Services/PulseAudioManager.vala

```diff
@@ -560,8 +560,40 @@ public class Sound.PulseAudioManager : GLib.Object {
 
             // Fallback to form_factor
             if (device.icon_name == null && device.form_factor != null) {
-                if (device.form_factor == "headphone") {
-                    device.icon_name = "audio-headphones";
+                switch (device.form_factor) {
+                    case "car":
+                        device.icon_name = "audio-car";
+                        break;
+                    case "computer":
+                    case "internal":
+                        device.icon_name = "computer";
+                        break;
+                    case "handset":
+                        device.icon_name = "phone";
+                        break;
+                    case "hands-free":
```

Comment:
It's a `hands-free` more a `headset` in that it has a microphone?

Reply:
Yes, you're correct! Thanks for catching that